### PR TITLE
List Mile (MILE)

### PR DESCRIPTION
**Pull request by vlad-khramov**
- Official project URL: http://mile.global/
- Official block explorer URL: https://explorer.mile.global/

### CODE REVIEW

File: assets/src/main/java/bisq/asset/MileAddressValidator.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import org.bitcoinj.core.Base58;
+
+import java.util.Arrays;
+import java.util.zip.CRC32;
+import java.util.zip.Checksum;
+
+/**
+ * Mile address - base58(32 bytes of public key + 4 bytes of crc32)
+ */
+public class MileAddressValidator implements AddressValidator {
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/MileAddressValidator.java]: Make this inner class of Mile as it will not be used anywhere else.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

File: assets/src/main/java/bisq/asset/MileAddressValidator.java (hunk 1 above)
Comment (vlad-khramov) [reply assets/src/main/java/bisq/asset/MileAddressValidator.java]: Fixed

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**